lib/utils/constants.dart

```diff
@@ -1,43 +1,46 @@
+import 'package:flutter/material.dart';
+
 class Constants {
-  static const imageBackdrop =
-      "https://image.tmdb.org/t/p/w500/9n2tJBplPbgR2ca05hS5CKXwP2c.jpg";
-  static const imagePoster =
-      "https://image.tmdb.org/t/p/w500/qNBAXBIQlnOThrVvA6mA2B5ggV6.jpg";
+  static const baseUrl = "https://image.tmdb.org/t/p/w500/";
+  static const imageBackdrop = "c9n2tJBplPbgR2ca05hS5CKXwP2c.jpg";
+  static const imagePoster = "qNBAXBIQlnOThrVvA6mA2B5ggV6.jpg";
 
-  static const ranking = '7.8';
-  static const releaseDate = "Release Date: 2023-04-05";
-  static const titleMovieCard = "The Super Mario Bros. Movie";
-  static const genreAnimation = "Animation";
-  static const genreFamily = "Family";
-  static const genreAdventure = "Adventure";
-  static const genreFantasy = "Fantasy";
-  static const genreComedy = "Comedy";
-  static const textOverview = "Overview";
-  static const textOverviewInfo =
-      "While working underground to fix a water main, Brooklyn plumbers—and brothers—Mario and Luigi are transported down a mysterious pipe and wander into a magical new world. But when the brothers are separated, Mario embarks on an epic quest to find Luigi.";
-  static const textOriginalTitle =
-      "Original Title: The Super Mario Bros. Movie";
+  static const overview = "Overview:";
+  static const originalTitle = "Original Title:";
   static const textLikeMovie = "Enjoy this movie? Give it a Like!";
 
   static const heightPercentageSizedBoxStack = 0.53;
+  static const widthPercentageBoxDecoration = 0.3;
   static const percentageSizedBox = 0.6;
   static const positionRankingContainer = 10.0;
   static const imagePositionTopSizedBox = 170.0;
 
+  static const backgroundColor = 0x0ff00133;
+  static const colorWhite = Colors.white;
+  static const colorYellow = Colors.yellow;
+  static const colorTransparent = Colors.transparent;
+  static const colorBlack26 = Colors.black26;
+  static const colorBlack54 = Colors.black54;
```

Comment:
This kind of constants colors should not be added as a const, because actually these are const

Reply:
Okay, fixed!